app/gui2/src/components/GraphEditor.vue

```diff
@@ -51,6 +54,29 @@ const interactionBindingsHandler = interactionBindings.handler({
   click: (e) => (e instanceof MouseEvent ? interaction.handleClick(e) : false),
 })
 
+// This is where the component browser should be placed when it is opened.
+function targetComponentBrowserPosition() {
+  const editedInfo = graphStore.editedNodeInfo
+  const isEditingNode = editedInfo != null
+  const hasNodeSelected = nodeSelection.selected.size > 0
+  const nodeSize = new Vec2(0, 24)
```

Comment:
I'm not sure I understand this hardcoded value. This is the expected size of a node that is about to be created by the component browser? If so, it should be loaded from a predefined constant. Let's change that soon, but in separate PR to not block anything.